libraries/botbuilder-core/tests/transcriptStoreBaseTest.js

```diff
@@ -103,214 +101,248 @@ exports._badArgs = function _badArgs(store) {
+            store.getTranscriptActivities('test', conversationId).then((pagedResult) => {
                 assert.equal(pagedResult.items.length, activities.length);
                 // delete!
                 store.deleteTranscript('test', conversationId).then(() => {
                     return Promise.all([
                         // check deleted
-                        store.getTranscriptActivities('test', conversationId).then(pagedResult => {
+                        store.getTranscriptActivities('test', conversationId).then((pagedResult) => {
                             assert.equal(pagedResult.items.length, 0);
                         }),
                         // check second transcript still exists
-                        store.getTranscriptActivities('test', conversationId2).then(pagedResult => {
+                        store.getTranscriptActivities('test', conversationId2).then((pagedResult) => {
                             assert.equal(pagedResult.items.length, activities2.length);
-                        })
-                    ])
-                })
+                        }),
+                    ]);
+                });
             }),
             // test B
-            store.getTranscriptActivities('test', conversationId2).then(pagedResult => {
+            store.getTranscriptActivities('test', conversationId2).then((pagedResult) => {
                 assert.equal(pagedResult.items.length, activities2.length);
-            })
+            }),
         ]);
     });
-}
+};
 
 exports._getTranscriptActivities = function _getTranscriptActivities(store, useParallel = true) {
     return new Promise((resolve, reject) => {
-        var conversationId = '_getTranscriptActivitiesPaging';
-        var date = new Date();
-        var activities = createActivities(conversationId, date, 50);
+        const conversationId = '_getTranscriptActivitiesPaging';
+        const date = new Date();
+        const activities = createActivities(conversationId, date, 50);
         // log in parallel batches of 10
-        var groups = group(activities, 10);
-        return promiseSeq(groups.map(group => () => resolvePromises(group.map(item => () => store.logActivity(item)), useParallel)))
-        .then(async () => {
-            var actualPageSize = 0;
-            var pagedResult = {};
-            var seen = [];
-            do {
-                pagedResult = await store.getTranscriptActivities('test', conversationId, pagedResult.continuationToken);
-                assert(pagedResult);
-                assert(pagedResult.items);
+        const groups = group(activities, 10);
+        return promiseSeq(
+            groups.map((group) => () =>
+                resolvePromises(
+                    group.map((item) => () => store.logActivity(item)),
+                    useParallel
+                )
+            )
+        )
+            .then(async () => {
```

Comment:
The then and catch is indented too much, it would be nicer if the .then is aligned with the parenthesis.

Reply:
I believe there are a few more cases

Reply:
Prettier mark this as a warning.